misago/core/testproject/urls.py

```diff
@@ -27,74 +27,74 @@
         name="django-i18n",
     ),
     # django-simple-sso doesn't have namespaces, we can't use namespace here
-    url(r"^sso/", include("misago.sso.urls")),
-    url(r"^forum/test-pagination/$", views.test_pagination, name="test-pagination"),
-    url(
-        r"^forum/test-pagination/(?P<page>[1-9][0-9]*)/$",
+    path("sso/", include("misago.sso.urls")),
+    path("forum/test-pagination/", views.test_pagination, name="test-pagination"),
+    re_path(
+        "forum/test-pagination/(?P<page>[1-9][0-9]*)/",
```

Comment:
It get's int with valid page number. `0123` is not valid page number

Reply:
if it is finally converted to number leading zero doesn't matter, even if it will be 0000123 finally as an int is 123. Do we need care of leading zeros? I would use `<int:page>` instead this regexp.